code/__DEFINES/mobs.dm

```diff
@@ -303,9 +303,9 @@
 #define OFFSET_NECK "neck"
 
 //MINOR TWEAKS/MISC
-#define AGE_MIN 17 //youngest a character can be
+#define AGE_MIN 18 //youngest a character can be
 #define AGE_MAX 85 //oldest a character can be
-#define AGE_MINOR 20  //legal age of space drinking and smoking
+#define AGE_MINOR 21  //legal age of space drinking and smoking
```

Comment:
the raising minimum age to 18 is fine but dont change drinking age. theres not any good reason for it.

Reply:
It keeps the age range for space drinking and smoking the same which i like